PendingReleaseNotes

```diff
@@ -3,6 +3,25 @@
 * RGW: GetObject and HeadObject requests now return a x-rgw-replicated-at
   header for replicated objects. This timestamp can be compared against the
   Last-Modified header to determine how long the object took to replicate.
+* ceph df detail output has got META section in plain format:
```

Comment:
CCing @zdover23 for this!

Reply:
s/a x-rgw-replicated-at/an x-rgw-replicated-at/

Reply:
@zdover23  - that's completely unrelated to this PR. To be fixed separately.